modules/iam-assumable-roles-with-saml/variables.tf

```diff
@@ -1,9 +1,11 @@
 variable "provider_name" {
   description = "Name of the SAML Provider"
+  type        = "string"
```

Comment:
There is no need in `type = "string"`. Once this module is converted to support Terraform 0.12 we may review this and add types, if necessary.